Uplift/Models/Gym.swift

```diff
@@ -253,6 +253,29 @@ struct FacilityDetail {
             subfacilities = detailData.subFacilityNames.compactMap({ $0 })
         }
     }
+
+    func getEquipmentCategories() -> [EquipmentCategory] {
+       var equipmentDict = [String: [Equipment]]()
```

Comment:
Is something up with the spacing here? I think `var` is only indented three spaces from the beginning of `func`?